src/types/value/types.ts

```diff
@@ -4,6 +4,7 @@ import type { ContentRelationshipField } from "./contentRelationship";
 import type { DateField } from "./date";
 import type { EmbedField } from "./embed";
 import type { GeoPointField } from "./geoPoint";
+import { GroupField } from "./group";
```

Comment:
```suggestion
import type { GroupField } from "./group";
```

Reply:
Oops! Thanks, I saw this suggestion from your other review but forgot to apply it.